src/Dashboard/Dashboard/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 0.3.0
+* Update swagger version to 2025-08-01 to include dashboard operations.
```

Comment:
- Remove `## Version 0.3.0` and put changelog under '## Upcoming Release'
- Use ‘Updated’ instead of 'Update'